Attendance/Projects/App/Sources/Di/Extension+AppDIContainer.swift

```diff
@@ -11,10 +11,10 @@ import DiContainer
 import UseCase
 
 extension AppDIContainer {
-  public func registerDefaultDependencies() async {
+   func registerDefaultDependencies() async {
     await registerDependencies { container in
-      var repositoryFactory = RepositoryModuleFactory()
-      let useCaseFactory = UseCaseModuleFactory()
+      var repositoryFactory = self.repositoryFactory
+      let useCaseFactory = self.useCaseFactory
       
       repositoryFactory.registerDefaultDefinitions()
```

Comment:
```suggestion
      self.repositoryFactory.registerDefaultDefinitions()
      let repositoryFactory = self.repositoryFactory
      let useCaseFactory = self.useCaseFactory
```
`registerDefaultDefinitions`를 먼저 호출하고 나서 repositoryFactory를 Immutable하게 사용하는 방식이 더 좋아 보입니다!

내부 구현을 모두 다 확인해보진 않았지만, `registerDefaultDefinitions` 메서드의 내부를 살펴보니 `repositoryDefinitions` 프로퍼티만 변경하는 것으로 확인됩니다.
혹시 해당 메서드 안에서만 사용하고 원본에 영향을 주지 않는게 의도하신거라면 그대로 유지하는게 맞는 것 같습니다!

Reply:
@honghoker  수정되었습니다